synapseclient/table.py

```diff
@@ -440,21 +440,27 @@ def _csv_to_pandas_df(
 
     line_terminator = str(os.linesep)
 
+    pandas_args = {
+        "dtype": dtype,
+        "sep": separator,
+        "quotechar": quote_char,
+        "escapechar": escape_char,
+        "header": 0 if contain_headers else None,
+        "skiprows": lines_to_skip,
+        **kwargs,
```

Comment:
Is this line necessary with the next line?

Reply:
Nope, removed it in the latest commit.